releasenotes/notes/feat-electronic-denisty-initializations-d1bd5131c5776418.yaml

```diff
@@ -0,0 +1,19 @@
+---
+features:
+  - |
+    Three news means of initializing an :class:`~qiskit_nature.second_q.properties.ElectronicDensity`
+    have been added.
```

Comment:
In the text here could you list the 3 capabilities. I can presume they are for an empty density, an identity and one based on the number of particles from the example code, but I think it would be nice to have some text here describing things rather than having to try to figure what the 3 things are from the code example - even though I grant you its not that hard. And maybe mention they all can optionally include the 2-body RDM or not, by default they do (as per the first 2 examples but not in the latter).

And for this text `Three news means of initializing...` the `initializing` has me thinking about constructors. Maybe `Three new methods for creating instances of...` or something like that.